Python/Product/Cookiecutter/CookiecutterToolWindow.cs

```diff
@@ -209,21 +209,19 @@ sealed class CookiecutterToolWindow : ToolWindowPane, IVsInfoBarUIEvents {
             _uiShell.UpdateCommandUI(0);
         }
 
-        private void OpenGeneratedFolder(string folderPath) {
-#if DEV15_OR_LATER
-            OpenInSolutionExplorer(folderPath);
-#else
-            OpenInWindowsExplorer(folderPath);
+        private void ExecuteCommand(string name, string args) {
+            try {
+                _dte.ExecuteCommand(name, args ?? string.Empty);
```

Comment:
I remember we had an issue with invoking new project from existing code a while back.  That was probably when our command names were a mess, before you cleaned them all up.
I'll test it again and I'll ask on the extensibility list.  I'd like proof of an actual issue before forcing template authors into using guids.